test/E2ETest/TyeRunTests.cs

```diff
@@ -11,10 +11,12 @@
 using System.Text.Json;
 using System.Text.Json.Serialization;
 using System.Threading.Tasks;
+using Microsoft.Diagnostics.Tracing.Parsers.JScript;
 using Microsoft.Tye;
 using Microsoft.Tye.Hosting;
 using Microsoft.Tye.Hosting.Model;
 using Microsoft.Tye.Hosting.Model.V1;
+using Org.BouncyCastle.Crypto;
```

Comment:
Why do we need this dependency?

Reply:
🤷 - did I put that there? will cleanup the unneeded usings for sure